docs/pds4_create_collection_product.rst

```diff
@@ -0,0 +1,60 @@
+``pds4_create_collection_product`` Program
+=================================
+
+Introduction
+------------
+The RMS Node's PDS4 Collection Product Creation Tool (``pds4_create_collection_product``)
+is designed to create bespoke collection product files from
+`Planetary Data System (PDS) <https://pds.nasa.gov>`_ `PDS4-format
+<https://pds.nasa.gov/datastandards/documents/>`_ label files. This tool automates the
+creation of collection products by recursively searching for all label files within a
+specified directory and extracting their identifying information. This information is then
+compared against user-provided details (the bundle and collection names) to determine
+which label files are primary members and which are secondary members. The results are
+compiled into a CSV file and placed in the user's specified location, or in the
+collection's location if no specific destination is provided. This allows for a quick and
+efficient way to check the contents of collections.
+
+**NOTE**: This tool is intended to be used on pre-existing label files. This tool does not
+include/create label files for collections. The use of this tool does not assign the Ring
+Moon Systems Node the responsibility of creating collections for the data provider. This
+tool also does not generate the associated collection product label file. All data
+providers are advised to work with their assigned Node to create the collection product
+label file.
+
+
+Command Line Arguments
+----------------------
+
+Required arguments
+^^^^^^^^^^^^^^^^^^
+
+Three command line arguments are required in every run.
+
+The first is the path to the collection you wish to make a collection product for. This
+path will also be referenced as the output path for the collection product if it is not
+otherwise specified.
+
+The second is the name of the bundle.
```

Comment:
This needs to be updated with the `--bundle` and `--collection` arguments, and put them below under the optional arguments. There is only one required argument.